sdk/docs/manually-written/sdk/tutorials/setup-assistant/installation.rst

```diff
@@ -136,6 +136,36 @@ Download Manually
 
 If you want to verify the SDK download for security purposes before installing, you can look at :doc:`our detailed instructions for manual download and installation <manual-download>`.
 
+Troubleshooting
+***************
+
+The Daml installation requires around 1.5 GB of disk space. If there is not enough space on disk, the installer outputs the error:
+``
+resource exhausted (No space left on device)
+``
+
+In order to fix this problem, one needs to ensure sufficient disk space overall, and in particuloar that the user's home directory and the temporary
+files directory have sufficient space (i.e., in the order of 2GiB each):
+
+* On Windows, one can find the user's home directory in the `%USERPROFILE%` environment variable, and the temporary files directory in the
+  ``%TEMP%`` environment variable
+* On Linux and MacOS, one can find the user's home directory in the ``HOME%`` environment variable, and the temporary files directory in the
+  ``TMPDIR`` environment variable 
+
+If the installer cannot ensure sufficient space in the the user's home directory and the temporary files directory, an alternative is to change
```

Comment:
```suggestion
If the installer cannot ensure sufficient space in the Daml SDK's home directory (``DAML_HOME``) and the temporary files directory, an alternative is to change
```

Depending on how you word the section before this one, just saying the name of the environment variable may be more appropriate.